spec/_widget/components/search.spec.js

```diff
@@ -123,6 +123,21 @@ describe('Search', () => {
         "SSL Certificates with Heroku": 5,
         "SSL Certificates with Windows": 6,
         "Getting Started with SSL Certificates": 7
+      },
+      'metrics': {
```

Comment:
In case it's not clear, these are automated tests that ensure that we don't degrade our rankings when we make a change to the widget/algorithm.